apps/src/StudioApp.js

```diff
@@ -394,6 +395,11 @@ StudioApp.prototype.init = function (config) {
         var nonDropletError = false;
         // are we trying to toggle from blocks to text (or the opposite)
         var fromBlocks = this.editor.currentlyUsingBlocks;
+        var event = fromBlocks ? "block-to-text" : "text-to-block";
+        firehoseClient.putRecord(
+          'analysis-events',
+          {study: 'block-and-text-transitions', event: event}
+        );
```

Comment:
Though it does seem weird, I don't think anything needs to necessarily be changed. It seems appropriate to me to store data for analysis in Firehose and data on errors in New Relic.